### PR TITLE
Feat: Persisted Queries

### LINKED ISSUE #93: Persisted Queries
I would like to discuss if we want to support persisted queries in the future. I like how relay handles it and personally i find them quite powerful in terms of security and saving bytes on network requests:

[https://relay.dev/docs/guides/persisted-queries/](https://relay.dev/docs/guides/persisted-queries/)

### PR DESCRIPTION
**Pull request by pixelmund**
I've gone ahead and wrote some docs. Feel free to adjust them however you like or i can do it too. 
I tested the apollo style in the example and it works smoothly. 

Closes #93

### CODE REVIEW

File: packages/houdini-common/src/config.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export class Config {
 	schema: graphql.GraphQLSchema
 	apiUrl?: string
 	schemaPath?: string
+	outputPath?: string
```
Comment (AlecAivazis) [inline packages/houdini-common/src/config.ts:41]: Can we rename this to something like `persistedQueryPath`? `outputPath` is pretty generic and will likely confuse me down the line.

File: README.md [hunk 2]
```diff
@@ -932,6 +933,92 @@ export default new Environment(async function ({ text, variables = {} }, session
 })
 ```
 
+## 🚦&nbsp;&nbsp;Persisted Queries
```
Comment (AlecAivazis) [inline README.md:936]: Thanks for adding this!

**Review by AlecAivazis — CHANGES_REQUESTED** [review]
Overall this looks pretty good! One small tweak/nitpick and we should be able to merge this in really quickly 👍

File: README.md (hunk 2 above)
Comment (pixelmund) [reply README.md:936]: :)

**Review by AlecAivazis — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**